content/tutorials/coordination.md

```diff
@@ -0,0 +1,122 @@
+
+### Requirements
+
+- If in sim, complete the [quickstart tutorial](https://mushr.io/tutorials/quickstart/)
+- If on real car, complete the [first steps tutorial](https://mushr.io/tutorials/first_steps/)
+- MuSHR Navigation System [mushr_navigation_system](https://mushr.io/tutorials/mushr_navigation_system/)
+
+## Coordination Planner Problem Statement
+
+There is a team of `num_agents` MuSHR cars and a set of `num_goals` tasks. Each task requires one car to follow all of the `num_waypoints` points and stop at the last one. The environment is a grid world where the movement of the car is restricted to orthongonal movement. This approximate approach can work well when combined with the [nhttc controller](https://github.com/naughtyStark/nhttc_ros). The coordination planner has the ability to create an initial path for each car that is avoiding all possible collisions. This path is leveraged with the nhttc controller to adapt a car's path and avoid collisions in real-time if need be. The combination of these two services is more powerful than just the nhttc controller by itself as potential conflicts are sorted out prior to declaring each car's path. It is possible that some cars will not have any tasks and others will have multiple to complete sequentially.
+
+## Install
+Clone repo and install the package: 
+{{< highlight bash >}}
+$ cd ~/catkin_ws/src/
+$ git clone https://github.com/prl-mushr/mushr_coordination.git
+$ cd mushr_coordination
+$ git clone https://github.com/whoenig/libMultiRobotPlanning.git
+$ cd ~/catkin_ws/
+$ catkin_make
+{{< / highlight >}}
+
+## API
+For adjusting params see `launch/mushr_coordination.launch` it has the environment params for the planner. Change the number of waypoints for each goal by setting parameter `num_waypoint`. Change the car team setting by editing or creating new config file. 
+
+## Running the coordination planner
+
+Run the planner that will set up the environment publishers and subscribers.
+{{< highlight bash >}}
+$ roslaunch mushr_coordination mushr_coordination.launch cars_file:={car's team config file}
+{{< / highlight >}}
+
+For simulation, run `rviz` then and subscribe to planner's topics. Download the [rviz config file](/tutorials/coordination_planner/nhttc.rviz) for an out-of-box rviz setup. This setup subscribes to all the important rostopics and organizes them in categories. 
+{{< highlight bash >}}
+$ rviz
+{{< / highlight >}}
+
+Below is the format for setting the initial position for a team of cars and a set of tasks to complete. For out-of-box examples, look at the **Example of Initializing the Planner** section which utilizes files already part of the repository.
+{{< highlight bash >}}
+//format for launching
+$ roslaunch mushr_coordination init_planner.launch cars_file:={car's team config file} task_file:={task benchmark file}
```

Comment:
`init_planner.launch` currently recognizes `tasks_file` as an arg, not `task_file`, so it doesn't launch with the proper task file when using this format.
```suggestion
$ roslaunch mushr_coordination init_planner.launch cars_file:={car's team config file} tasks_file:={task benchmark file}
```